components/widgets/menu/menu_wrapper_animation.jsx

```diff
@@ -38,11 +35,9 @@ export default class MenuWrapperAnimation extends React.PureComponent {
 
     onExiting = (node) => {
         node.style.overflowY = 'hidden';
-        const maxHeight = Math.min(node.scrollHeight + 10, ((window.innerHeight - 70) * 80) / 100);
         const nodeStyler = styler(node);
         composite({
-            maxHeight: tween({from: maxHeight, to: 0, duration: ANIMATION_DURATION}),
-            opacity: chain(delay(ANIMATION_DURATION / 2), tween({from: 1, to: 0, duration: ANIMATION_DURATION / 2})),
+            opacity: chain(delay(0), tween({from: 1, to: 0, duration: ANIMATION_DURATION})),
```

Comment:
here you don't need to specify the `chain`or the `delay(0)` because it is exactly the same than just the `tween({from: 1, to: 0, duration: ANIMATION_DURATION})` directly.

Reply:
That's what I thought, but it does not work as expected without the delay.

Reply:
You can try it yourself locally.

Reply:
Going to check it because it doesn't make sense at all

Reply:
I tested it and works for me, actually, it can be simplified way more, you can have exactly the same effect with this code:
```js
    onEntering = (node) => {
        const nodeStyler = styler(node);

        tween({from: 0, to: 1, duration: ANIMATION_DURATION}).start({
            update: (v) => nodeStyler.set({opacity: v}),
        });
    }

    onExiting = (node) => {
        const nodeStyler = styler(node);
        tween({from: 1, to: 0, duration: ANIMATION_DURATION}).start({
            update: (v) => nodeStyler.set({opacity: v}),
        });
    }
```

Reply:
or even better:
```js
    onEntering = (node) => {
        const nodeStyler = styler(node);
        tween({from: {opacity: 0}, to: {opacity: 1}, duration: ANIMATION_DURATION}).start(nodeStyler.set);
    }

    onExiting = (node) => {
        const nodeStyler = styler(node);
        tween({from: {opacity: 1}, to: {opacity: 0}, duration: ANIMATION_DURATION}).start(nodeStyler.set);
    }
```